### ISSUE TITLE
Julia crashes when fitting a SVC

### ISSUE THREAD

**Issue by flo-he** [alan-turing-institute/MLJ.jl#1030]
**Describe the bug**
When running the example code for [training a SVC on the iris dataset](https://alan-turing-institute.github.io/MLJ.jl/dev/models/SVC_LIBSVM/#Using-a-built-in-kernel), the julia kernel crashes shortly after execution of the `fit!` function.

**To Reproduce**
```julia
using MLJ
import LIBSVM

SVC = @load SVC pkg=LIBSVM
model = SVC(kernel=LIBSVM.Kernel.Linear)

X, y = @load_iris
mach = machine(model, X, y) |> fit!
```

**Output:**
```bash
[ Info: For silent loading, specify `verbosity=0`.
import MLJLIBSVMInterface ✔
MLJLIBSVMInterface.SVC

SVC(
  kernel = LIBSVM.Kernel.Linear,
  gamma = 0.0,
  cost = 1.0,
  cachesize = 200.0,
  degree = 3,
  coef0 = 0.0,
  tolerance = 0.001,
  shrinking = true)

((sepal_length = [5.1, 4.9, 4.7, 4.6, 5.0, 5.4, 4.6, 5.0, 4.4, 4.9  …  6.7, 6.9, 5.8, 6.8, 6.7, 6.7, 6.3, 6.5, 6.2, 5.9], sepal_width = [3.5, 3.0, 3.2, 3.1, 3.6, 3.9, 3.4, 3.4, 2.9, 3.1  …  3.1, 3.1, 2.7, 3.2, 3.3, 3.0, 2.5, 3.0, 3.4, 3.0], petal_length = [1.4, 1.4, 1.3, 1.5, 1.4, 1.7, 1.4, 1.5, 1.4, 1.5  …  5.6, 5.1, 5.1, 5.9, 5.7, 5.2, 5.0, 5.2, 5.4, 5.1], petal_width = [0.2, 0.2, 0.2, 0.2, 0.2, 0.4, 0.3, 0.2, 0.2, 0.1  …  2.4, 2.3, 1.9, 2.3, 2.5, 2.3, 1.9, 2.0, 2.3, 1.8]), CategoricalArrays.CategoricalValue{String, UInt32}["setosa", "setosa", "setosa", "setosa", "setosa", "setosa", "setosa", "setosa", "setosa", "setosa"  …  "virginica", "virginica", "virginica", "virginica", "virginica", "virginica", "virginica", "virginica", "virginica", "virginica"])

[ Info: Training machine(SVC(kernel = Linear, …), …).
```
After that, Julia just crashes.

**Versions**
<details>

Julia Version 1.9.1
Commit 147bdf428c (2023-06-07 08:27 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: 16 × 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, tigerlake)
  Threads: 1 on 16 virtual cores

MLJ v0.19.2

</details>

**Comment by ablaom** [comment]
The issue is most likely https://github.com/JuliaML/LIBSVM.jl/issues/95 , specific to Windows and a Julia, not MLJ/LIBSVM issue. It is apparently fixed by Julia 1.9.2 still in testing.

**Comment by ablaom** [comment]
Pretty sure this is resolved in 1.9.3. Feel free to reopen if not.
